5/README.md

```diff
@@ -22,46 +28,76 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 ## Tango Property Specification
 
- A Property is designed to represent any information of configuration in Tango.
+This specification is intended to formally document the Tango Property model.
+Runtime representation of Property in conforming Tango implementations MUST follow this specification.
 
 ### Goals
 
- The specification of Property aims to define a standard structure and type to use in any implementation of Tango.
+A Property represents a configuration parameter in Tango.
 
-Additionally, it aims to:
-
-* Provide the initial information when one starts a Tango Device
-
-* Be usable as keeping global variable in the Tango control system
-
-* Provide meta data complementing any data in Tango
+There are 4 kinds of Tango Properties:
 
+ *  Class Property: a Property related to a Device Class (cf RFC-2).
+ *  Device Property: a Property related to a Device (RFC-2).
+ *  Attribute Property: a Property related to an Attribute (RFC-4).
+ *  Free Property: a Property defined for the entire Control System (RFC-1) i.e not related to any Class, Device or Attribute.
 
 ### Use Cases
 
-There are many use cases for the usage of a Property:
-
-* To connect a real network equipment a Property can define the target IP address
-
-* To change the representation of an information in order to be more user friendly a Property can define a new data format for the graphical user interface to convert the raw data.
+Here are some Tango Property use cases:
 
+* To connect a real network equipment a Property can define the target IP address or hostname
+* To change the representation of an information in order to be more user friendly a Property can define a new data format for the graphical user interface to convert the raw data
+* To configure some alarms on Attribute Value (See RFC-4)
+* To store the Attribute value of a Memorized Attribute (See RFC-4)
+* To configure the labels associated to a Tango:DevEnum attribute values
+* ...
 
 ## Specification
 
-A Tango Property is a strict definition of a pair of key/value
-* configuration are represented in the form of properties.
-* data are represented in the form of attributes.
-* actions are represented in the form of commands.
-
-Its model can be represented as a defined tree which each elements are from a defined types: Class, Device, Property, Attribute, Command following the rules below:
-
-
-A Tango Property is a strict definition of a pair of key/value
-* The Property SHALL have one key, called Property Name
-* The Property SHALL have one value
+A Tango Property is a strict definition of a pair of key/value:
+  * The Property SHALL have one key, called Property Name
+  * The Property SHALL have one value, which could be empty, called Property Value
+
+A Tango Property MAY be persisted in the Tango Database of the concerned Tango Control System (See RFC-1 and RFC-6) or into a file if no Tango Database is used.
```

Comment:
Suggest to remove "of the concerned Tango Control System" as implicitly implied...
Moreover, is it really the case a Property MAY be persisted? Being configuration parameters cannot see much value in not saving a Property... nor a Property Value.

Reply:
We can imagine having these device properties being passed as argin parameters.

Reply:
...mmmh... of course it's feasible, but is it really desirable for Tango? Not sure about this, but it makes me somehow uncomfortable...

Reply:
> ...mmmh... of course it's feasible, but is it really desirable for Tango? Not sure about this, but it makes me somehow uncomfortable...

It could be useful for tests for instance.

Reply:
> A Tango Property MAY be persisted....

The idea here as well is that the property is not always persisted.
If you define a Default property value and this default is reasonable for your current use case for all of your devices, you don't need to persist the property value into the database.